dmapiclient/base.py

```diff
@@ -50,6 +52,10 @@ def iter_method(self, *args, **kwargs):
 
 
 class BaseAPIClient(object):
+    RETRIES = 3
+    RETRIES_BACKOFF_FACTOR = 0.3
+    RETRIES_FORCE_STATUS_CODES = (500, 502, 503, 504)
```

Comment:
A comment to explain what `RETRIES_FORCE_STATUS_CODES` does?

Reply:
Yep, I'll add one. It's a list of HTTP status codes on which to force a retry. Default is none.